static/app/components/modals/dashboardWidgetLibraryModal/utils.tsx

```diff
@@ -0,0 +1,13 @@
+const WIDGET_LIBRARY_VISITS = 'dashboard-widget-library-visits';
+
+export function shouldShowNewBadge(): boolean {
+  const visits = localStorage.getItem(WIDGET_LIBRARY_VISITS);
+  return visits === null || (parseInt(visits, 10) || 0) < 5;
+}
+export function setWidgetLibraryVisit() {
+  const visits = localStorage.getItem(WIDGET_LIBRARY_VISITS);
+  localStorage.setItem(
+    WIDGET_LIBRARY_VISITS,
+    visits === null ? '0' : `${(parseInt(visits, 10) || 0) + 1}`
```

Comment:
```suggestion
    visits === null ? '1' : `${(parseInt(visits, 10) || 0) + 1}`
```
Should this be 1 instead of 0 for the first click?